pocs/utils/messaging.py

```diff
@@ -46,25 +46,20 @@ class PanMessaging(object):
       * telemetry:commands (in ArduinoIO... new)
       * camera:commands (in ArduinoIO... new)
 
-    And some other channels are used in tests:
+    And some other topics are used in tests:
 
-      * TEST-CHANNEL (test_messaging.py)
+      * Test-Topic (test_messaging.py)
```

Comment:
Seems like we now have a mix of naming conventions. I know they are all allowed but should we have some logical scheme?

Reply:
That change was deliberate, just to test that the title-case form was supported.